lib/util.js

```diff
@@ -589,6 +589,9 @@ function formatValue(ctx, value, recurseTimes, ln) {
       if (keyLength === 0)
         return ctx.stylize(`[${name}]`, 'special');
       base = `[${name}]`;
+    } else if (types.isModuleNamespaceObject(value)) {
+      braces[0] = `[${tag}] {`;
+      formatter = formatNamespaceObject;
```

Comment:
It would be nice to move this down to the last else because it is probably one of the least used values and does not need to be performant.